standalone/content/js/standaloneRoomViews.jsx

```diff
@@ -522,6 +597,73 @@ loop.standaloneRoomViews = (function(mozL10n) {
     }
   });
 
+  var StandaloneNotificationView = React.createClass({
```

Comment:
ESLint is saying StandaloneNotificationView is defined and never used.   Likely just needs to be added to the return statement on line 971 to fix. After fixing, run ESLint again and fix errors found there.